includes/polling/unix-agent/packages.inc.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+$pkgs_id=[];
+$pkgs_db_id=[];
```

Comment:
and spaces, please. Like everywhere else in code.
[Code-Guidelines](https://docs.librenms.org/Developing/Code-Guidelines/)